packages/api/src/api.ts

```diff
@@ -11,18 +14,18 @@ interface TendermintConnection {
  * Options to pass into the RegenApi constructor.
  */
 export interface RegenApiOptions {
-  connection: TendermintConnection;
+  connection: ConnectionOptions;
 }
 
 /**
  * The main entry point for interacting with the Regen Ledger. The class needs
  * a client connection
  */
 export class RegenApi {
-  readonly connection: Connection;
+  readonly client: ProtobufRpcClient;
 
-  constructor(connection: Connection) {
-    this.connection = connection;
+  constructor(client: ProtobufRpcClient) {
+    this.client = client;
```

Comment:
I think we should rename it to `queryClient` because this won't be the same for txs

Reply:
If the API can return a queryClient or a txClient, shouldn't we keep this the more generic `client`? Maybe you are picturing a different way to connect to a txClient? We can discuss monday.

Reply:
using queryClient now